files/helm/fluent-operator/common.yaml

```diff
@@ -1,6 +1,13 @@
 containerRuntime: containerd
 operator:
   priorityClassName: system-cluster-critical
+  resources:
+    limits:
+      cpu:
+      memory:
```

Comment:
lets set values here. its too much black magic and not so clear without understanding how yaml handles no value maps